backend/blueprint/api/config/urlConfig.py

```diff
@@ -0,0 +1,2 @@
+# Configuration variable for the application
+BASE_URL = 'http://badgeyay-api.herokuapp.com/'
```

Comment:
When we are testing locally, we will always have to change this to localhost, and again before commiting we have to change it back.
On other hand, if we hard code it to localhost here, auto deployment will cause trouble.
Do we have any other alternative here?

Reply:
I will try to use the debug config as locally we will set it as true but not in the production. What do you think?

Reply:
yes sounds good :+1: